docs/src/concepts/index_ivfpq.md

```diff
@@ -0,0 +1,78 @@
+# Understanding LanceDB's IVF-PQ index
+
+An index is a data structure that represents data in a way that makes it more efficient to search and retrieve. Using an index is faster, but less accurate than kNN or brute force search because, in essence, the index is a lossy representation of the data.
+
+LanceDB is fundamentally different from other vector databases in that it is built on top of [Lance](https://github.com/lancedb/lance), an open-source columnar data format designed for performant ML workloads and fast random access. Due to the design of Lance, LanceDB's indexing philosophy adopts a primarily *disk-based* indexing philosophy.
+
+## IVF-PQ
+
+IVF-PQ is a composite index that combines inverted file index (IVF) and product quantization (PQ). The implementation in LanceDB provides several parameters to fine-tune the index's size, query throughput, latency and recall, which are described later in this section.
+
+### Product quantization
+
+Quantization is a compression technique used to reduce the dimensionality of an embedding to speed up search. 
+
+![](../assets/ivfpq_quantization.webp)
+
+Product quantization (PQ) works by dividing a large, high-dimensional vector of size into equally sized subvectors. Each subvector is assigned a "reproduction value" that maps to the nearest centroid of points for that subvector. The reproduction values are then assigned to a codebook using unique IDs, which can be used to reconstruct the original vector.
+
+![](../assets/ivfpq_pq_desc.webp)
```

Comment:
Again, it's unclear what each of these symbols mean. Perhaps you can illustrate the example described below concretely.

Here's a sketch of how I might do it:

![pq_diagram](https://github.com/lancedb/lancedb/assets/5488879/6f2eecf0-7c94-4be3-97ec-974328e91ee9)

Reply:
Great point, addressed in the new version!